python/ray/serve/_private/deployment_state.py

```diff
@@ -176,6 +181,7 @@ def __init__(
         controller_name: str,
         replica_tag: ReplicaTag,
         deployment_name: str,
+        scheduling_strategy="SPREAD",
```

Comment:
Does this guarantee that each node will get one GRPC driver?

Reply:
type hint here